### ISSUE TITLE
Concurrent createUploadSession requests failing

### ISSUE THREAD

**Issue by OlliV** [OneDrive/onedrive-api-docs#1360]
#749 
## Category
- [ ] Question
- [ ] Documentation issue
- [X] Bug

#### Expected or Desired Behavior

I have been using `createUploadSessions` in SPO for months now and it has worked perfectly for uploading a large chunk of files. What I normally do is that I spin up 40 concurrent requests, start uploading the file chunks, and start new sessions once any of the previous ones is finished. This has worked fine until now, the sessions were created, chunks were created and finally the files were created in OneDrive.

#### Observed Behavior

What I'm seeing now that I receive an `invalidRequest` response after creating a bunch of requests and it seems like only a handful of files will get uploaded completely. I can make it work by only creating one session, finishing it, and creating another session. However, this is considerably slower that what it used to be when I was able to upload multiple files concurrently.

```
  method: 'POST',
  path: '/sites/root/drive/root:%2F<snip>.pdf:/createUploadSession',
  responseBody: '{"error":{"code":"invalidRequest","message":"Invalid request","innerError":{"date":"2020-09-06T14:53:11","request-id":"dff3cfa9-54e8-4eb5-b108-bf0dec0b04e6"}}}'
```

If this is a new rate limit being applied, I believe the error code should be changed to something more meaningful or understandable.

#### Steps to Reproduce

Create a bunch of upload sessions concurrently and start uploading chunks. The specific code I'm using is located here:
https://github.com/turist-cloud/ship/tree/master/packages/ship-board

- `src/upload-files.ts`
- `src/fetch-graph-api.ts`

[ ]: http://aka.ms/onedrive-api-issues
[x]: http://aka.ms/onedrive-api-issues

**Comment by chackman** [comment]
The attribution seems like a bug, which we'll put on our backlog. To be clear, we discourage spinning up that many concurrent requests - the application should be getting a throttling response.

**Comment by OlliV** [comment]
@chackman Oh yes, I was going to say that I was hoping for a throttling response. I'm not sure if that happened before but my code would handle that just fine.

**Comment by patrick-rodgers** [comment]
As part of a repository clean up effort we are closing older issues. If this issue remains, please: open a new issue, reference this issue, and provide any additional details that may help in resolution. Thank you for your understanding as we work to improve our responsiveness.
